### PR TITLE
fix(VSparkline): incorrect bar lengths

### LINKED ISSUE #7762: [Bug Report] Wrong bar heights in sparkline
### Versions and Environment
**Vuetify:** 2.0.0-beta.5
**Last working version:** 2.0.0-beta.4
**Vue:** 2.6.10
**Browsers:** Chrome 75.0.3770.100
**OS:** Windows 10

### Steps to reproduce
create a bar sparkline with [1,2] values

### Expected Behavior
bar heights should correspond with values

### Actual Behavior
they don't

### Reproduction Link
beta.5: https://codepen.io/anon/pen/OeozZg?editors=1000

### Other comments
beta.4: https://codepen.io/anon/pen/KjxxoR?editors=1000

### PR DESCRIPTION
**Pull request by nekosaur**
## Description
sparkline showed incorrect bar lenghts when all values were larger than zero

## Motivation and Context
fixes #7762

## Markup:

<details>

```vue
<template>
  <div>
    <div class="ma-2 red">
      <v-sparkline :value="[-1, 0, 1]" type="bar" show-labels></v-sparkline>
    </div>
    <div class="ma-2 green">
      <v-sparkline :value="[1, 2]" type="bar" show-labels></v-sparkline>
    </div>
    <div class="ma-2 blue">
      <v-sparkline :value="[-1, -2]" type="bar" show-labels></v-sparkline>
    </div>
    <div class="ma-2 yellow">
      <v-sparkline :value="[1, 1]" type="bar" show-labels></v-sparkline>
    </div>
  </div>
</template>

```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/components/VSparkline/helpers/core.ts [hunk 1]
```diff
@@ -32,7 +32,9 @@ export function genBars (
   const { minX, maxX, minY, maxY } = boundary
   const totalValues = values.length
   const maxValue = Math.max(...values)
-  const minValue = Math.min(...values)
+  let minValue = Math.min(...values)
+
+  if (minValue > 0) minValue = 0
```
Comment (jacekkarczmarczyk) [inline packages/vuetify/src/components/VSparkline/helpers/core.ts]: `const minValue = Math.min(0, ...values)`

File: packages/vuetify/src/components/VSparkline/helpers/core.ts [hunk 2]
```diff
@@ -32,7 +32,9 @@ export function genBars (
   const { minX, maxX, minY, maxY } = boundary
   const totalValues = values.length
   const maxValue = Math.max(...values)
-  const minValue = Math.min(...values)
+  let minValue = Math.min(...values)
```
Comment (sh7dm) [inline packages/vuetify/src/components/VSparkline/helpers/core.ts]:
```suggestion
  let minValue = Math.min(0, ...values)
```

Comment (sh7dm) [reply packages/vuetify/src/components/VSparkline/helpers/core.ts]: (c) jacek

Comment (jacekkarczmarczyk) [reply packages/vuetify/src/components/VSparkline/helpers/core.ts]: const*

**Pull request merged**